docs/measurements/procedures/rigid_waist_shift.md

```diff
@@ -25,11 +83,11 @@
 
 ## Procedure Per IP
 
-This procedure is applied at every experimental insertion (IR1, IR2, IR5, IR8) where local corrections need to be established or checked.
-Keep in mind that this does Beam1 and Beam2 at the same time, but different IPs cannot/ should not be done in parallel.
+This procedure is applied at the main experimental insertions (IR1 and IR5) where local corrections need to be established or checked.
+Keep in mind that this does Beam1 and Beam2 at the same time, but different IPs cannot / should not be done in parallel.
 
 !!! warning "Orbit Feedback"
-      Please remember to **always** keep the orbit feedback system **ON** during this procedure.
+      Please remember to **always** keep the orbit feedback system **ON** during this procedure if crossing angles are present in the machine.
```

Comment:
if you could explain what happens otherwise, that would be great

Reply:
We're not sure? Technically it would not be an issue for the machine, but could mess up the setup we need to find good corrections. Maybe this precision was not necessary.

Reply:
Simplified in [14abe46](https://github.com/pylhc/pylhc.github.io/pull/74/commits/14abe462aac917a757b1b6b3d51852e686029608)